dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/security/oidc/GenericOidcProviderBuilderConfigParserTest.java

```diff
@@ -91,13 +96,18 @@ public void parseConfigMissingRequiredParameter()
         Properties properties = new Properties();
 
         properties.put( "oidc.provider.idporten.client_id", "testClientId" );
-        properties.put( "oidc.provider.idporten.client_secret", "testClientSecret!#!?" );
-        properties.put( "oidc.provider.idporten.token_uri", "https://oidc-ver2.difi.no/token" );
-        properties.put( "oidc.provider.idporten.user_info_uri", "https://oidc-ver2.difi.no/userinfo" );
-        properties.put( "oidc.provider.idporten.jwk_uri", "https://oidc-ver2.difi.no/jwk" );
-        properties.put( "oidc.provider.idporten.end_session_endpoint", "https://oidc-ver2.difi.no/endsession" );
-
-        List<Map<String, String>> parse = GenericOidcProviderConfigParser.parse( properties );
+        properties.put( "oidc.provider.idporten.client_secret",
+            "testClientSecret!#!?" );
+        properties.put( "oidc.provider.idporten.token_uri",
+            "https://oidc-ver2.difi.no/token" );
+        properties.put( "oidc.provider.idporten.user_info_uri",
+            "https://oidc-ver2.difi.no/userinfo" );
+        properties.put( "oidc.provider.idporten.jwk_uri",
+            "https://oidc-ver2.difi.no/jwk" );
+        properties.put( "oidc.provider.idporten.end_session_endpoint",
+            "https://oidc-ver2.difi.no/endsession" );
```

Comment:
I am wondering why this got moved to next line - I think it should fit in a line by our formatting rules. Maybe you have shorter line length configured? In this case its simply more readable in a single line.

Reply:
You're totally right. I commented it out and the autoformatter did the comment formatting that has much shorter line width.